### PR TITLE
dragonball: support dragonball tracing for runtime-rs.

### PR DESCRIPTION
**Pull request by lisongqian**
This PR supports dragonball tracing for runtime-rs.

Fixes: #7249

Signed-off-by: Songqian Li <mail@lisongqian.cn>

### CODE REVIEW

**Comment by lisongqian** [comment]
Block on https://github.com/kata-containers/kata-containers/pull/7831.

File: src/runtime-rs/crates/runtimes/src/manager.rs [hunk 1]
```diff
@@ -107,12 +108,14 @@ impl RuntimeHandlerManagerInner {
             ) {
                 warn!(sl!(), "failed to setup tracing, {:?}", e);
             }
+            // tracing subscriber can share wich
```
Comment (ZizhengBian) [inline src/runtime-rs/crates/runtimes/src/manager.rs:111]: typo: tracing subscriber can share with hypervisor

Comment (lisongqian) [reply src/runtime-rs/crates/runtimes/src/manager.rs]: Fixed.

**Review by ZizhengBian — APPROVED** [review]
LGTM

**Pull request closed without merging**